src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: New Relic Kubernetes components
```

Comment:
Yes, we've created a new page for components and moved the content from this page to the components page because those components are the current ones.